### PR TITLE
Seperate `DiscordMember` database model

### PR DESCRIPTION
**Pull request by CarrotManMatt**
Merging this will make large changes to the database. Ensure a backup is made of all running deployments before changes are merged!

This change will improve database access speed, and reduce storage by not repeatedly storing hashed IDs multiple times

### CODE REVIEW

File: cogs/send_get_roles_reminders.py [hunk 1]
```diff
@@ -123,11 +123,8 @@ async def send_get_roles_reminders(self) -> None:
             if not member_requires_opt_in_roles_reminder:
                 continue
 
-            hashed_member_id: str = SentGetRolesReminderMember.hash_member_id(member.id)
             sent_get_roles_reminder_member_exists: bool = (
-                await SentGetRolesReminderMember.objects.filter(
-                    hashed_member_id=hashed_member_id,
-                ).aexists()
+                await SentGetRolesReminderMember.objects.filter(discord_id=member.id).aexists()  # type: ignore[misc]
```
Comment (MattyTheHacker) [inline cogs/send_get_roles_reminders.py:127]: image

ruh roh

**Review by MattyTheHacker — CHANGES_REQUESTED** [review]

File: cogs/send_get_roles_reminders.py (hunk 1 above)
Comment (CarrotManMatt) [reply cogs/send_get_roles_reminders.py:127]: ohhhhh, i know how to solve this.

**Review by MattyTheHacker — CHANGES_REQUESTED** [review]
`RuntimeError: You cannot use AsyncToSync in the same thread as an async event loop - just await the async function directly.`

**Review by MattyTheHacker — APPROVED** [review]
epic

**Pull request merged**